src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="POST"
+      data-qa="big">
```

Comment:
```suggestion
      data-qa="big"
    >
```
Closing bracket should be on the new line